dev/src/components/CounterComponent.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import T from 'prop-types';
```

Comment:
T isn't the best naming convention... clean code principles encourage self explanatory names.... can we revert back to PropTypes (this applies to all files throughout review)?